<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugWindow</class>
 <widget class="QDialog" name="DebugWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отладка</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="Text">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="NumberTable">
       <property name="text">
        <string>ХТ продавцов по номеру телефона</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="NumberTree">
       <property name="text">
        <string>Дерево конспектов по номеру телефона</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="NameTree">
       <property name="text">
        <string>Дерево продавцов по ФИО</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="PriceTree">
       <property name="text">
        <string>Дерево продацов по цене</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="ThemeTree">
       <property name="text">
        <string>Дерево конспектов по теме</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="DispTree">
       <property name="text">
        <string>Дерево конспектов по дисциплине</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="AddressTree">
       <property name="text">
        <string>Дерево продавцов по месту жительства</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
